openapi/p4pa-pu-bff.openapi.yaml

```diff
@@ -706,6 +745,47 @@ components:
         number:
           type: integer
           format: int64
+    ReceiptDetailDTO:
+      type: object
+      required:
+        - receiptId
+        - paymentAmountCents
+        - remittanceInformation
+        - debtPositionDescription
+        - debtorFullName
+        - debtorFiscalCode
+        - pspCompanyName
+        - iud
+      properties:
+        receiptId:
+          type: integer
+          format: int64
+        iuv:
+          type: string
+        paymentAmountCents:
+          type: integer
+          format: int64
+        remittanceInformation:
+          type: string
+        debtPositionDescription:
+          type: string
+        debtorFullName:
+          type: string
+        debtorFiscalCode:
+          type: string
+        payerFullName:
+          type: string
+        payerFiscalCode:
+          type: string
```

Comment:
what about use a common DTO, as PersonDTO of Receipt, to nest in a common DTO these info introducing also here just debtor and payer fields on the root of the json?